UnitsNet.Tests/CustomCode/HeatFluxTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated (once) by \generate-code.bat, but will not be
+//     regenerated when it already exists. The purpose of creating this file is to make 
+//     it easier to remember to implement all the unit conversion test cases.
+// 
+//     Whenever a new unit is added to this quantity and \generate-code.bat is run,
+//     the base test class will get a new abstract property and cause a compile error
+//     in this derived class, reminding the developer to implement the test case
+//     for the new unit.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add Extensions\MyQuantityExtensions.cs to decorate quantities with new behavior.
+//     Add UnitDefinitions\MyQuantity.json and run GeneratUnits.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Copyright (c) 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com).
+// https://github.com/angularsen/UnitsNet
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+using System;
+using Xunit;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class HeatFluxTests : HeatFluxTestsBase
+    {
+        protected override double BtuPerSquareFootHourInOneWattPerSquareMeter => 3.16998331e-1;
+        protected override double BtuPerSquareFootMinuteInOneWattPerSquareMeter => 5.28330551e-3;
+        protected override double BtuPerSquareFootSecondInOneWattPerSquareMeter => 8.80550918e-5;
+        protected override double BtuPerSquareInchSecondInOneWattPerSquareMeter => 6.11493693e-7;
+        protected override double CalPerSquareCentiMeterSecondInOneWattPerSquareMeter => 2.39005736e-5;
```

Comment:
I think I prefer typing these fully out, `Calories` and `Kilocalories`.

And again, you have two variants `Calorie (IT)` and `Calorie (th)`. 

No idea what they are, but they keep showing up:
http://www.unitconversion.org/unit_converter/heat-flux-density-ex.html